data/wesionary-team.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "wesionaryTEAM Co. Ltd.",
+    "establishedAt": 1643853528,
```

Comment:
The time equivalent to this timestamp seems to be `Feb 3rd 2022`. Is it the actual time?
If I look at the website the established Date is `1580667300` (Feb 3rd 2020).

Please confirm 🙏

Reply:
The date seems to be 3rd of February according to the company website
![image](https://user-images.githubusercontent.com/60746512/200098210-ce49810c-9fd9-42b3-96df-848c991ee201.png)
@bibek-magar Can you make changes?

Reply:
Thank you .. done